app/helpers/application_helper/toolbar/hosts_center.rb

```diff
@@ -188,7 +190,9 @@ class ApplicationHelper::Toolbar::HostsCenter < ApplicationHelper::Toolbar::Basi
           :url_parms => "main_div",
           :confirm   => N_("Shutdown the selected items?"),
           :enabled   => false,
-          :onwhen    => "1+"),
+          :onwhen    => "1+",
+          :klass   => ApplicationHelper::Button::HostFeatureButtonWithDisable,
+          :options => {:feature => :shutdown}),
         button(
```

Comment:
@romanblanco, The culprit was @feature, not @record. I added an additional check to skip the extra logic if it is null. We need to have this in hosts_center too, to disable features that are not supported, and to enable only start and stop.